EESSI tutorial @ ISC'26 (#38)
EESSI/isc-tutorial: isc26-EESSI/attendance.tex

Based on attendance at previous EESSI and EasyBuild events, including the yearly EasyBuild User Meetings (over 50
attendees in the last physical edition in 2025, close to 100 in 2021 and 2022 in an online setting) and previous
related tutorials (PRACE-VI-SEEM 2017 Spring School with 25-30 attendees, a free online EasyBuild tutorial in June'20
with about 100 attendees, the half-day ISC'21 tutorial with over 25 attendees, the in-person (half-day) ISC'22
EasyBuild tutorial with about 20 attendees, and an online EESSI tutorial in Dec'23 had over 130 registrations) plus a 

> recent webinar series related to EESSI in May-June 2025 with 50 attendees each , we

Author: ```suggestion
recent webinar series related to EESSI in May-June 2025 with over 150 people who registered to attend, we
```

> anticipate this EESSI tutorial to be well attended with 20-50 attendees.

Author: ```suggestion
Based on attendance at previous EESSI and EasyBuild events, including:
\begin{itemize}
\item recent webinar series related to EESSI in May-June 2025 with over 150 registrations;
\item the yearly EasyBuild User Meetings (over 50 in-person attendees, alongside over 70 online registrations, in 2025);
\item an online EESSI tutorial in Dec'23 with over 130 registrations;
\item the in-person (half-day) ISC'22 EasyBuild tutorial with about 20 attendees;
\end{itemize}
we anticipate this EESSI tutorial to be well attended with 20-50 attendees.

Based on attendance at previous EESSI and EasyBuild events, including:
\begin{itemize}
\item recent webinar series related to EESSI in May-June 2025 with over 150 registrations;
\item the yearly EasyBuild User Meetings (over 50 in-person attendees, alongside over 70 online registrations, in 2025);
\item an online EESSI tutorial in Dec'23 with over 130 registrations;
\item the in-person (half-day) ISC'22 EasyBuild tutorial with about 20 attendees;
\end{itemize}

> we anticipate this EESSI tutorial to be well attended with 20-50 attendees.

Reviewer: ```suggestion
We anticipate this EESSI tutorial at ISC to be well attended with 20-50 attendees. This estimate is based on attendance at previous EESSI and EasyBuild events, including:
\begin{itemize}
\item recent webinar series related to EESSI in May-June 2025 with over 150 registrations;
\item the yearly EasyBuild User Meetings (over 50 in-person attendees, alongside over 70 online registrations, in 2025);
\item an online EESSI tutorial in Dec'23 with over 130 registrations;
\item the in-person (half-day) ISC'22 EasyBuild tutorial with about 20 attendees.
\end{itemize}
```
